src/components/header/header.jsx

```diff
@@ -1,22 +1,50 @@
 import React from 'react'
 import './header.css';
-import { Link } from 'react-router-dom'
+import { NavLink, } from 'react-router-dom'
+
+
+
+const navigationLinks = [
+  {
+    id: 1,
+    title: 'home',
+    route: '/',
+  },
+  {
+    id: 2,
+    title: 'Calculator',
+    route: '/calculator',
+  },
+  {
+    id: 3,
+    title: 'Qoutes',
+    route: '/quotes',
+  },
+];
+
 
 function header() {
   return (
     <>
       <div className="App">
         <nav className="App-header">
           <ul className="App-header-container" >
-            <li>
-              <Link to="/">Home</Link>
-            </li>
-            <li>
-              <Link to="/Calculations">Calculations</Link>
-            </li>
-            <li>
-              <Link to="/Qoutes">Qoutes</Link>
-            </li>
+            {navigationLinks.map(({ id, title, route }) => (
+              <li key={id}>
+                <NavLink
+                  to={route}
+                  style={({ isActive }) => ({
+                    color: isActive ? '#fff' : '#545e6f',
+                    background: isActive ? '#7600dc' : '#f0f0f0',
+                    borderRadius: isActive ? '2px' : '',
+                  })}
+                >
+
+                  {title}
+                </NavLink>
+              </li>
+            ))}
+
           </ul>
         </nav >
       </div >
```

Comment:
- [ ] Please, you have bags to fix to get your project displayed so that we can check if the general layout matches the wireframes in the[ project requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md)
- Here is the feedback when you ran the app locally. 
![1](https://user-images.githubusercontent.com/63972670/177432857-66d798dc-a876-40f8-bd14-a8adf5bf914b.PNG)
- Try to fix it, and submit it again